tests/scripts/thread-cert/node.py

```diff
@@ -445,6 +492,18 @@ def set_partition_id(self, partition_id):
         self.send_command(cmd)
         self._expect('Done')
 
+    def get_pollperiod(self):
+        self.send_command('pollperiod')
+        i = self._expect(r'(\d+)\r?\n')
+        if i == 0:
+            pollperiod = self.pexpect.match.groups()[0]
+        self._expect('Done')
+        return pollperiod
```

Comment:
`pollperiod` might be unassigned here.

Reply:
If it is not explicitly specified by the user, it would be the default keep-alive period derived from ChildTimeout. See
https://github.com/openthread/openthread/blob/master/src/core/api/link_api.cpp#L163
https://github.com/openthread/openthread/blob/master/src/core/mac/data_poll_sender.cpp#L190

That is, it should always have return value.

Reply:
In that case we don't need this check:
`if i == 0:`

Reply:
agree. will update.

Reply:
noticing it comes from #4325 @bukepo

Reply:
from pexpect's doc,`0` means success. I think we should use `assert()` instead of `==` for this . But since this is an common issue in this file, I tend to fix it all in a separate PR.

Reply:
@bukepo agree.